lib/services/ios-debug-service.ts

```diff
@@ -103,11 +103,18 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
 
 		let lineStream = byline(child_process.stdout);
 		this._childProcess = child_process;
+		const pidRegExp = new RegExp(`${debugData.applicationIdentifier}:\\s?(\\d+)`);
```

Comment:
I'm not sure where's the appropriate place for this code. I agree that it needs tests and I'll add them in a separate PR.